api/nodes/serializers.py

```diff
@@ -150,13 +151,17 @@ def create(self, validated_data):
         auth = Auth(user)
         node = self.context['view'].get_node()
         pointer_node = Node.load(validated_data['node']['_id'])
-        pointer = node.add_pointer(pointer_node, auth, save=True)
-        return pointer
+        if not pointer_node:
+            raise exceptions.ParseError('Node not found.')
+        try:
+            pointer = node.add_pointer(pointer_node, auth, save=True)
+            return pointer
+        except ValueError:
+            raise exceptions.ParseError('Pointer to node {} already in list'.format(pointer_node._id))
```

Comment:
Although this does return a `400` response, "ParseError" doesn't accurately describe the error. 

Is there an alternative way to return a 400 response? Possibly a custom DRF exception? @brianjgeiger may have thoughts on this.